packages/server/src/health/service.ts

```diff
@@ -39,6 +41,7 @@ export class HealthService extends Service {
     this.table = new HealthTable()
     this.watcher = new HealthWatcher(this.conduitService, this.instanceService, this)
     this.logger = this.loggerService.root.sub('health')
+    this.webhookBroadcaster = new WebhookBroadcaster(this.configService, this.webhookService)
```

Comment:
`webhookBroadcaster` is kind on unnecessarily long. You don't have really anything else here that could be thought of as a broadcaster. I would just name this variable `broadcaster`